Example/Segment-Adjust.xcodeproj/project.pbxproj

```diff
@@ -248,7 +248,11 @@
 				LastUpgradeCheck = 0730;
 				ORGANIZATIONNAME = "Prateek Srivastava";
 				TargetAttributes = {
+					6003F589195388D20070C39A = {
+						DevelopmentTeam = ENLA8E7Z4L;
```

Comment:
What is this string? Will it work on everyone's environments?

Reply:
It's under the Segmentio org

Reply:
How is it setup on the CI and everyone else's computer?